src/functions.php

```diff
@@ -0,0 +1,54 @@
+<?php
+require_once 'src/db_structures.php';
+//function that converts the dogs array into its individual fields from the db_functions
+function returnDogs(array $dog_data): string
+{
+    $dog_string = '';
+
+    foreach ($dog_data as $dog) {
+        if ($dog['owner_id'] == 2) {
+            $belongs = 'I belong to Finn';
+
+        } else {
+            $belongs = 'I belong to Nan';
+        }
```

Comment:
Doesn't the owner information come from the database? What if someone wants to own a pet that isn't Finn or Nan?